src/applications/edu-benefits/sass/edu-benefits.scss

```diff
@@ -140,6 +140,24 @@
   list-style-type: none !important;
 }
 
+.wizard-button::after {
+  background-image: url(/img/arrow-down-white.svg);
+  background-position: right bottom;
+  background-repeat: no-repeat;
+  background-size: 0.6em auto;
+  content: "";
+  display: inline-block;
+  height: 1em;
+  margin-left: 0.5em;
+  margin-right: -0.3em;
+  text-indent: -9999em;
+  width: 1em;
+}
+
+.wizard-button[aria-expanded=true]:after {
+  background-image: url(/img/arrow-up-white.svg);
+}
+
```

Comment:
These css additions should probably be added to the platform styles since it'll be used by all wizards eventually.